core/src/main/resources/hudson/tasks/Fingerprinter/config.properties

```diff
@@ -0,0 +1,24 @@
+# The MIT License
+#
+# Copyright (c) 2019, Mit Naria <mit4dev@gmail.com>
```

Comment:
Removing copy-pasted copyright from other file. Same goes for other license files.